### DISCUSSION TITLE
Shared Gpu memory

### DISCUSSION THREAD

**Discussion by thelockster101** [carson-katri/dream-textures#666] [Ideas]
I run out of vram when generating images, I have an 8gb vram card and 32gb of ram "So that makes 16gb of shared memory".
My Gpu is an rx 6600, Is there a fix I am unaware of? I can't see why it can't use shared memory.

**Comment by NullSenseStudio** [comment]
Shared memory cannot be used for generating. To reduce VRAM usage you can use memory optimizations such as manual attention slice size of 1 or CPU offloading.
